Sources/ArcGISToolkit/Components/AR/FlyoverSceneView.swift

```diff
@@ -104,7 +131,12 @@ public struct FlyoverSceneView: View {
         SceneViewReader { sceneViewProxy in
             sceneViewBuilder(sceneViewProxy)
                 .cameraController(cameraController)
-                .onAppear { session.start() }
+                .onAppear {
+                    if shouldOrientToCompass {
+                        session.configuration.worldAlignment = .gravityAndHeading
+                    }
+                    session.start()
+                }
```

Comment:
Now that the view needs to know about the configuration, I think it would be cleaner to just pass the configuration to the start method. Then the configuration property on 163 goes away.